Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,107 @@
+//
+//  File.m
+//
+//
+//  Created by Dante Tam on 6/10/21.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "AMPStorage.h"
+
+@interface AMPStorage ()
+@end
+
+@implementation AMPStorage {
```

Comment:
No implementation?

Reply:
The braces are unnecessary, I just realized.

Reply:
That's the old way for declaring properties inside curly bracket.